website/widgets/form/table.md

```diff
@@ -285,11 +285,12 @@ needed. For tables with > 10 rows, the initial height is `500px`.
 In some cases though, it might be desirable to control the dimensions yourself via `width` and `height`
 attributes. Both accept [CSS units](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units),
 however `%` values for height might not work as you think (especially in [flex layout](/docs/layout#flex-layout))
-so we discourage its use in favor of more static units like `px` or `rem`.
+so we discourage its use in favor of more static units like `px` or `rem`. If you wish to use all of the available height,
```

Comment:
Let's add a new paragraph with a separate `height='1'` example rather than changing the existing one.